source/_integrations/dynalite.markdown

```diff
@@ -127,17 +176,29 @@ dynalite:
       port: 12345
       autodiscover: true
       polltimer: 1
-      areacreate: auto
-      log_level: debug
       area:
+        '1':
+          name: Office
         '2':
           name: Living Room
+          nodefault: true
           channel:
             '2': 
               name: Entrance Spot
               fade: 10.0
             '3': 
               name: Dining Table
+	      preset:
```

Comment:
Please replace the `TAB`s used here by spaces.

Reply:
done